extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/HttpConfiguration.java

```diff
@@ -49,6 +49,20 @@
     @ConfigItem(defaultValue = "8444")
     public int testSslPort;
 
+    /**
+     * If this is true then the address, scheme etc will be set from headers forwarded by the proxy server, such as
+     * {@code X-Forwarded-For}. This should only be set if you are behind a proxy that sets these headers.
+     */
+    @ConfigItem(defaultValue = "false")
+    public boolean proxyAddressForwarding;
```

Comment:
Why isn't it the default to use those values if present? Security reasons because any user could use them?

Reply:
Yea, any client can add them if this is not configured and spoof their address etc.

Reply:
OK thanks.